GAMBLR_examples_output.log

```diff
@@ -361,10 +358,10 @@ capture  genome
 +     include_hotspots = FALSE, genome_build = "hg38", these_samples_metadata = fl_meta)
 
 > dim(coding_tabulated_df)
-[1] 784 116
```

Comment:
I don't understand why this went from 116 genes to 111. @Kdreval do you have any thoughts?